### PR TITLE
Add a docker_users variable

### PR DESCRIPTION
**Pull request by jamesla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ Docker Compose installation options.
 
 (Used only for RedHat/CentOS.) You can enable the Edge or Test repo by setting the respective vars to `1`.
 
+Set the docker_users variable to allow those users to run docker (these users are added to the docker group)
+
+    docker_users:
+      - Guy
```
Comment (gramosbr) [inline README.md]: Who's Guy? 😄

**Comment by DidelotK** [comment]
I think you should replace the task `Add users to docker group` by this:
```yml
- name: Add users to docker group.
  user:
    name: "{{ item }}"
    groups: docker
    append: yes
  with_items: "{{ docker_users }}"
```
This will add docker user to docker group instead of setting primary group to user.

**Comment by jamesla** [comment]
@DidelotK Good spotting have updated to append groups rather than set them. 

Changing to include_tasks will require some work because the tests fail (I can see there is another PR sitting there with failed tests). From a quick look it looks like the version of ansible needs to be updated on the test containers so that it can find the include_tasks module so I will leave that to the person with a PR open already.

**Comment by jamesla** [comment]
bump @geerlingguy

File: README.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ Docker Compose installation options.
 
 (Used only for RedHat/CentOS.) You can enable the Edge or Test repo by setting the respective vars to `1`.
 
+Set the docker_users variable to allow those users to run docker (these users are added to the docker group)
```
Comment (geerlingguy) [inline README.md]: This comment should appear below the variable (to follow the convention with the other vars in the README).

File: README.md (hunk 1 above)
Comment (geerlingguy) [inline README.md]: Heh, can you just use for an example:

```yaml
    docker_users:
      - jane_doe
      - john_doe
```

A little more generic-ish.

File: defaults/main.yml [hunk 3]
```diff
@@ -17,3 +17,6 @@ docker_apt_repository: "deb https://download.docker.com/linux/{{ ansible_distrib
 docker_yum_repo_url: https://download.docker.com/linux/centos/docker-{{ docker_edition }}.repo
 docker_yum_repo_enable_edge: 0
 docker_yum_repo_enable_test: 0
+
+# Docker users
```
Comment (geerlingguy) [inline defaults/main.yml]: Just to make it a little more clear: `A list of users who will be added to the docker group.`?

File: tasks/docker-users.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Add users to docker group
```
Comment (geerlingguy) [inline tasks/docker-users.yml]: Can you change to `Ensure docker users are added to the docker group.`?

**Comment by jamesla** [comment]
Changes made as requested, ready to be merged @geerlingguy

**Comment by jamesla** [comment]
@geerlingguy bump

**Comment by jamesla** [comment]
@geerlingguy bump

**Comment by abest0** [comment]
Drats... any reason why this was closed... it's a great addition.

**Comment by bartmeuris** [comment]
+1 for this

I currently do this with a separate role with a dependency on this-one - which works fine.

**Pull request merged**